src/man/firejail-profile.txt

```diff
@@ -467,6 +467,12 @@ available in the new namespace is a new loopback interface (lo).
 Use this option to deny network access to programs that don't
 really need network access.
 
+.SH Other
+.TP
+\fBjoin-or-start sandboxname
+Join the sandbox identified by name or start a new one.
+Same as "firejail --join=sandboxname" command if sandbox with specified name exists, otherwise same as "name sandboxname".
```

Comment:
Would `otherwise same as "name sandboxname"` be better as`otherwise same as "firejail --name=sandboxname"`?